src/main/pg/timerio.c

```diff
@@ -29,8 +29,16 @@
 
 #include "timerio.h"
 
-#define TIMER_PIN_MAP(idx, pin, pos, dmaOpt)  \
-        { config[idx].ioTag = IO_TAG(pin); config[idx].index = pos; config[idx].dmaopt = dmaOpt; }
+/*
+Details of the TIMER_PIN_MAP macro:
+    i => is the index in the PG timer IO config array,
+    p => is the hardware pin to be mapped,
+    o => is the hardware pin occurrence (1 based index) within timerHardware (full) that should be used
+                e.g. 1 = first occurence of the pin, 2 = second occurence, etc.
+    d => the configured dma opt for the pin
```

Comment:
this also needs to be better documented, perhaps with a reference to where the dma options are defined in the codebase.

I've been use to positive integer based DMA option previously, but the `timer map` command in this PR is outputting negative ones.

Reply:
-1 is not in use.